tools/ci.sh

```diff
@@ -141,6 +141,24 @@ function ci_esp8266_build {
 ########################################################################################
 # ports/nrf
 
+function ci_mimxrt_setup {
+    ci_gcc_arm_setup
+}
+
+function ci_mimxrt_build {
+    make ${MAKEOPTS} -C mpy-cross
+    make ${MAKEOPTS} -C ports/mimxrt submodules
+    make ${MAKEOPTS} -C ports/mimxrt BOARD=MIMXRT1010_EVK
+    make ${MAKEOPTS} -C ports/mimxrt BOARD=MIMXRT1020_EVK
+    make ${MAKEOPTS} -C ports/mimxrt BOARD=MIMXRT1050_EVK
+    make ${MAKEOPTS} -C ports/mimxrt BOARD=MIMXRT1060_EVK
+    make ${MAKEOPTS} -C ports/mimxrt BOARD=MIMXRT1064_EVK
+    make ${MAKEOPTS} -C ports/mimxrt BOARD=TEENSY40
```

Comment:
I'm not sure all boards need to be built, it slows down the CI.  Maybe just MIMXRT1010_EVK and TEENSY40?

Reply:
Surely we can reduce that. We would prefer MIMXRT1020 and TEENSY 40, because we use these two boards for testing.

Reply:
Sounds good.  I see this is added to the pins PR, so I will concentrate on that first.

Reply:
Hi Damien,
We have decoupled both PRs so you can choose which one you want to focus on. Nonetheless I would be happy if the pin PR gets merged first 😄.

Reply:
Mhmh I guess I will mention you in the PR right away so that you can see what I am working on. If you have points I could maybe spare extra work. 
The PR is still in draft mode but most of the work is already done. At least I hope it is 😂.